### PR TITLE
Hc flaky cuke fix 1

### PR DESCRIPTION
**Pull request by harshared**
### Master Redmine ticket
* (Required!)

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: features/hbx_admin/manage_sep_types/create_sep_type.feature [hunk 1]
```diff
@@ -40,18 +26,7 @@ Feature: Admin has ability to create a new SEP Type
...
-    And Admin fills Create SEP Type form with Title
-    And Admin fills Create SEP Type form with Event label
-    And Admin fills Create SEP Type form with Tool Tip
-    And Admin selects <market_kind> market radio button
-    And Admin fills Create SEP Type form with Reason
-    And Admin selects effective on kinds for Create SEP Type
-    And Admin <action> select termination on kinds for <market_kind> SEP Type
-    And Admin fills Create SEP Type form with Pre Event SEP and Post Event SEP dates
-    And Admin clicks on Create Draft button
+    When Admin creats new SEP Type with <market_kind> market and <action> select termination on kinds with failure scenario
```
Comment (bill-transue) [inline features/hbx_admin/manage_sep_types/create_sep_type.feature:29]: You misspelled `creates`.

File: features/step_definitions/employers_steps.rb [hunk 2]
```diff
@@ -442,7 +442,8 @@
 end
 
 When(/^.+ go(?:es)? to the benefits tab$/) do
-  find(".interaction-click-control-benefits").click
+  find(EmployerHomePage.benefits_link).click
+  sleep 5
```
Comment (bill-transue) [inline features/step_definitions/employers_steps.rb:446]: Let's not do sleep 5, we should instead add `wait: 5` to the next `find` so that it will only wait as long as needed.

File: features/hbx_admin/step_definitions/manage_sep_types_steps.rb [hunk 3]
```diff
@@ -335,6 +335,32 @@ def fehb_qualifying_life_events
   end
 end
 
+When(/Admin creats new SEP Type with (.*) market and (.*) select termination on kinds with (.*) scenario$/) do |market_kind, action, scenario|
```
Comment (bill-transue) [inline features/hbx_admin/step_definitions/manage_sep_types_steps.rb:338]: You'll need to fix `creats` here too.

File: features/hbx_admin/manage_sep_types/create_sep_type.feature [hunk 4]
```diff
@@ -9,26 +9,12 @@ Feature: Admin has ability to create a new SEP Type
     And the Admin will see the Manage SEPs under admin dropdown
     And Admin can click Manage SEPs link
 
-  Scenario Outline: Admin will create a new <market_kind> SEP type
+  Scenario Outline: Admin will <scenario> scenario create a new <market_kind> SEP type
```
Comment (bill-transue) [inline features/hbx_admin/manage_sep_types/create_sep_type.feature:12]: Why did you add `<scenario> scenario` to this step?

**Review by bill-transue — CHANGES_REQUESTED** [review]
Refactor looks ok overall, a few minor tweaks. Also need to remove the `sleep 5`s.

File: features/hbx_admin/manage_sep_types/sep_type_visibility.feature [hunk 5]
```diff
@@ -1,77 +1,34 @@
...
-    Given the Admin is on the Main Page
+    And all market kinds are enabled for user to select
+    And all announcements are enabled for user to select
+    And that a user with a HBX staff role with hbx_tier3 subrole exists
+    And Hbx Admin logs on to the Hbx Portal
+    And the Admin is on the Main Page
     And the FAA feature configuration is enabled
     And Qualifying life events of all markets are present
     And the Admin will see the Manage SEPs under admin dropdown
     And Admin can click Manage SEPs link
```
Comment (bill-transue) [inline features/hbx_admin/manage_sep_types/sep_type_visibility.feature:12]: These 3 steps seem unnecessary.

**Review by bill-transue — CHANGES_REQUESTED** [review]
We have some technical problems in the step definitions with sleeps we should fix. We also have some extra steps in the background of the one scenario that seem unnecessary. The last issue is there are still a lot of steps being done for creating the SEPs. I'm not sure we need to create them through the browser directly. Should we consider creating on the backend with helper methods?

File: features/hbx_admin/manage_sep_types/sep_type_visibility.feature (hunk 5 above)
Comment (harshared) [reply features/hbx_admin/manage_sep_types/sep_type_visibility.feature]: even though they seem like we are just checking, we are creating seps and clicking manage sep link dropdown

**Review by bill-transue — APPROVED** [review]
Great work @harshared, you nailed the Given refactor.
